src/app/static/src/tests/components/runWorkflow/runWorkflow.test.ts

```diff
@@ -119,67 +130,83 @@ describe(`runWorkflow`, () => {
             })
 
         expect(wrapper.find(runWorkflowCreate).vm.$data.runWorkflowMetadata).toMatchObject(workflowMetadata)
-        await wrapper.find("#clone").trigger("click")
-        expect(wrapper.find(workflowWizard).exists()).toBe(true)
+        wrapper.find("#clone").trigger("click")
 
-        expect(wrapper.vm.$data.workflowStarted).toBe(true)
-        expect(wrapper.find("#add-report-header").text()).toBe("Add reports")
+        setTimeout(async () => {
+            expect(wrapper.vm.$data.workflowStarted).toBe(true)
+            expect(wrapper.find(workflowWizard).exists()).toBe(true)
 
-        const buttons = wrapper.findAll("button")
-        expect(buttons.at(0).text()).toBe("Remove report")
-        expect(buttons.at(1).text()).toBe("Add report")
-        expect(buttons.at(2).text()).toBe("Cancel")
-        expect(buttons.at(3).text()).toBe("Next")
```

Comment:
I don't think removing this code is necessary except you are replacing it with selenium tests.

Reply:
It's still there, it's just moved down and inside the setTimeout